tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920120.yaml

```diff
@@ -27,7 +27,7 @@ tests:
 
               555-555-0199@example.com
               ----------397236876--
-            protocol: "http"
+            version: HTTP/1.0
```

Comment:
Well, i can change everything to 1.1 but i wasn't sure. We should do some recommendation on this. I prefer to use 1.1.

Reply:
By adding `HTTP/1.0` you've effectively changed the semantics of the tests since `1.1` is the default. It should always be 1.1, unless there's a very specific reason.